client/commands/v2/pysa_server.py

```diff
@@ -60,6 +65,56 @@ def __init__(
         self.binary_location = binary_location
         self.server_identifier = server_identifier
 
+    def invalid_model_to_diagnostic(
+        self,
+        invalid_model: query.InvalidModel,
+    ) -> lsp.Diagnostic:
+        return lsp.Diagnostic(
+            range=lsp.Range(
+                start=lsp.Position(
+                    line=invalid_model.line - 1, character=invalid_model.column
+                ),
+                end=lsp.Position(
+                    line=invalid_model.stop_line - 1,
+                    character=invalid_model.stop_column,
+                ),
+            ),
+            message=invalid_model.full_error_message,
+            severity=lsp.DiagnosticSeverity.ERROR,
+            code=None,
+            source="Pysa",
+        )
+
+    def invalid_models_to_diagnostics(
+        self,
+        invalid_models: Sequence[query.InvalidModel],
+    ) -> Dict[Path, List[lsp.Diagnostic]]:
+        result: Dict[Path, List[lsp.Diagnostic]] = {}
+        for model in invalid_models:
+            if model.path is None:
+                self.log_and_show_message_to_client(
+                    "Path cannot be None", lsp.MessageType.WARNING
```

Comment:
We should give the use the error message, rather than something generic like this. General rule: if you're not sure what's going on, give the user enough info so they can figure it out temselves.